src/app/app.module.ts

```diff
@@ -1,50 +1,52 @@
 // ngrx modules - START
-import {EffectsModule} from '@ngrx/effects';
-import {MetaReducer, Store, StoreModule} from '@ngrx/store';
-import {RouterStateSerializer, StoreRouterConnectingModule} from '@ngrx/router-store';
-import {StoreDevtoolsModule} from '@ngrx/store-devtools';
-import {storeFreeze} from 'ngrx-store-freeze';
-import {LoggerService} from './services/logger/logger.service';
-// enforces immutability
-export const metaReducers: MetaReducer<any>[] = !environment.production
-  ? [storeFreeze]
-  : [];
-// ngrx modules - END
-
-// APP store
-import { CustomSerializer, reducers } from './store/reducers';
-import { effects } from './store/effects';
-
+import { EffectsModule } from '@ngrx/effects';
+import { RouterStateSerializer, StoreRouterConnectingModule } from '@ngrx/router-store';
+import { MetaReducer, Store, StoreModule } from '@ngrx/store';
+import { StoreDevtoolsModule } from '@ngrx/store-devtools';
+import { storeFreeze } from 'ngrx-store-freeze';
+import { LoggerModule, NgxLoggerLevel } from 'ngx-logger';
+import { environment } from '../environments/environment';
+import { EnvironmentConfig, ENVIRONMENT_CONFIG } from '../models/environmentConfig.model';
 import { initApplication } from './app-initilizer';
-
-// common provider
-import { ProvidersModule } from './providers/providers.module';
 // app routes
 import { ROUTES, routingConfiguration } from './app.routes';
-import { SharedModule } from './shared/shared.module';
-import { LoggerModule, NgxLoggerLevel } from 'ngx-logger';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { MonitoringService } from './services/logger/monitoring.service';
-import { CryptoWrapper } from './services/logger/cryptoWrapper';
-import { AbstractAppInsights, AppInsightsWrapper } from './services/logger/appInsightsWrapper';
-import { DefaultErrorHandler } from './services/errorHandler/defaultErrorHandler';
+import { AppComponent } from './containers/app/app.component';
+// common provider
+import { ProvidersModule } from './providers/providers.module';
 import { AcceptTermsService } from './services/acceptTerms/acceptTerms.service';
-import { ExuiCommonLibModule, LAUNCHDARKLYKEY } from '@hmcts/rpx-xui-common-lib';
-import { PaymentLibModule } from '@hmcts/ccpay-web-component';
-import { ENVIRONMENT_CONFIG, EnvironmentConfig } from '../models/environmentConfig.model';
 import { CaseShareService } from './services/case/share-case.service';
+import { DefaultErrorHandler } from './services/errorHandler/defaultErrorHandler';
+import { AbstractAppInsights, AppInsightsWrapper } from './services/logger/appInsightsWrapper';
+import { CryptoWrapper } from './services/logger/cryptoWrapper';
+import { LoggerService } from './services/logger/logger.service';
+import { MonitoringService } from './services/logger/monitoring.service';
+import { SharedModule } from './shared/shared.module';
+import { effects } from './store/effects';
+// ngrx modules - END
+// APP store
```

Comment:
_Nitpick:_
This comment and the one above is no longer relevant, following the reordering of the imports.